community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/RelationshipTypeCheckerStep.java

```diff
@@ -60,16 +58,21 @@ public RelationshipTypeCheckerStep( StageControl control, Configuration config,
     @Override
     protected void process( Batch<InputRelationship,RelationshipRecord> batch, BatchSender sender ) throws Throwable
     {
+        Map<Object,MutableLong> types = typeCheckers.get( Thread.currentThread() );
```

Comment:
functions can be of cause extracted as constants;
or at least we can try to use map.computeIfAbsent or map.putIfAbsent to remove those if checks